components/prism-cmake.js

```diff
@@ -0,0 +1,18 @@
+	'boolean': /\b(ON|OFF|TRUE|FALSE)\b/,
+	'namespace': /\b(PROPERTIES|SHARED|PRIVATE|STATIC|PUBLIC|INTERFACE|TARGET_OBJECTS)\b/,
+	'operator': /\b(NOT|AND|OR|MATCHES|LESS|GREATER|EQUAL|STRLESS|STRGREATER|STREQUAL|VERSION_LESS|VERSION_EQUAL|VERSION_GREATER|DEFINED)\b/,
+	'comment': {
```

Comment:
Please move comment and string before all other tokens. This will solve the current issue with:

```
# ON OFF add_compile_options
"ON"
```

Also, please make it `'comment': /#.*/`. You don't need to create a new object.

Reply:
done

Reply:
You still need to reorder.

Reply:
Ok, sure I misunderstood that.

Reply:
Now it should be fine